code/modules/clothing/suits/marine_armor.dm

```diff
@@ -438,6 +438,8 @@ var/list/squad_colors_chat = list(rgb(230,125,125), rgb(255,230,80), rgb(255,150
 		/obj/item/device/walkman,
 	)
 
+	var/registered_mob
```

Comment:
hard ref gonna cause hard dels if someone is gibbed with a smartgun (this isn't nulled)